tests/tt_metal/tt_metal/unit_tests_fast_dispatch/command_queue/test_EnqueueProgram.cpp

```diff
@@ -1265,8 +1332,8 @@ TEST_F(CommandQueueFixture, TestRandomizedProgram) {
     for (uint32_t i = 0; i < NUM_ITERATIONS; i++) {
         auto rng = std::default_random_engine {};
         std::shuffle(std::begin(programs), std::end(programs), rng);
-        if (i % 10 == 0) {
-            log_debug(tt::LogTest, "Enqueueing {} programs for iter: {}/{} now.", programs.size(), i+1, NUM_ITERATIONS);
+        if (i % 50 == 0) {
+            log_info(tt::LogTest, "Enqueueing {} programs for iter: {}/{} now.", programs.size(), i+1, NUM_ITERATIONS);
```

Comment:
Do we want this to actually be log_info, or revert to log_debug?

Reply:
I like this as log_info, otherwise when I'm running tests I keep wondering if the test has hung